src/components/checkbox/checkbox.stories.svelte

```diff
@@ -18,6 +18,11 @@
       type: 'string',
       description: 'The gap between the label and the checkbox'
     },
+    '--leo-checkbox-flex-direction': {
+      control: 'select',
+      description: 'The flex direction of the label/checkbox',
+      options: ['row', 'row-reverse', 'column', 'column-reverse']
```

Comment:
I lean towards changing this to something like `--leo-checkbox-direction` or `--leo-checkbox-label-position` and not allowing `column` since it would be a strange, likely never used, look.